### DISCUSSION TITLE
Open FIA documents (pdf files) in app

### DISCUSSION THREAD

**Discussion by stefanverkerknl** [f1multiviewer/issue-tracker#201] [Feature requests] [answered]
Please add a feature that makes it possible to open FIA documents (pdf files) in the app instead of downloading it and opening it with a 3rd party app or (on Mac OS) in Quick Look. PDF.js offers a neat solution.

Thanks!

**Comment by LapsTimeOFF** [comment, marked as answer]
This has been added since a few versions.
